### PR TITLE
feat: DNS samples record sets

### PR DESCRIPTION
**Pull request by msampathkumar**
For Cloud DNS, add a terraform example to create a "A" type DNS record.

Terraform sample is tested.

If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-the-terraform-providers), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/main/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none
```

### CODE REVIEW

File: mmv1/products/dns/terraform.yaml [hunk 1]
```diff
@@ -27,6 +27,12 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         min_version: beta
         ignore_read_extra:
           - "force_destroy"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "dns_record_set_basic"
+        primary_resource_id: "default"
```
Comment (megan07) [inline mmv1/products/dns/terraform.yaml:32]: I think we're assuming the `primary_resource_id` would be for `google_dns_managed_zone` when we do the import. [This](https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-6173-old..auto-pr-6173#diff-40524a01cbca31921bea35a20c5fb30f7fae6a968e81337c5d581d71c1108530R130) is what it generates downstream and the cause of the test failure.

**Comment by msampathkumar** [comment]
Hi [megan07](https://github.com/megan07) so I only need this sample to be published to terraform-docs-sample b't not necessarily to Hasicorp pages. Can you also confirm if the `skip_docs: true` is fine? Many thanks.

**Review by megan07 — APPROVED** [review]
LGTM! Thanks!

**Comment by msampathkumar** [comment]
Hi @megan07 Thank you for your help 👍

**Pull request merged**